packages/kmod-6.1-nvidia/nvidia-fabricmanager.cfg

```diff
@@ -0,0 +1,34 @@
+# Modern, systemd-aware settings:
+# - Log to journal via stderr
+# - Keep running in the foreground
+LOG_LEVEL=4
+LOG_FILE_NAME=
+DAEMONIZE=0
+
+# Use Unix domain sockets instead of localhost ports.
+UNIX_SOCKET_PATH=/run/nvidia/fabricmanager.sock
+FM_CMD_UNIX_SOCKET_PATH=/run/nvidia/fabricmanager-cmd.sock
+
+# Start Fabric Manager in bare metal or full pass through virtualization mode.
+FABRIC_MODE=0
+FABRIC_MODE_RESTART=0
+
+# Terminate on NVSwitch and GPU config failure.
+FM_STAY_RESIDENT_ON_FAILURES=0
+
+# When there is a GPU to NVSwitch NVLink failure, remove the GPU with the failure
+# from NVLink P2P capability.
+ACCESS_LINK_FAILURE_MODE=0
+
+# When there is a NVSwitch to NVSwitch NVLink failure, exit Fabric Manager.
+TRUNK_LINK_FAILURE_MODE=0
+
+# When there is a NVSwitch failure or a NVSwitch is excluded, abort Fabric Manager.
+NVSWITCH_FAILURE_MODE=0
```

Comment:
nit: it is fine either way but [preferred English style](https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html) would be "an NVSwitch" because it starts with a vowel sound ("en" as in "envy").
```suggestion
# When there is an NVSwitch to NVSwitch NVLink failure, exit Fabric Manager.
TRUNK_LINK_FAILURE_MODE=0

# When there is an NVSwitch failure or an NVSwitch is excluded, abort Fabric Manager.
NVSWITCH_FAILURE_MODE=0
```

Reply:
changed as suggested